server/src/utils/BaseController.js

```diff
@@ -38,13 +38,20 @@ export class BaseController {
     }
   };
 
-  getMany = async (req, res, next) => {
-    let filter, sort, skip, limit;
+  getMany = async (req, res, next, query={}) => {
+    let filter,
+      sort,
+      skip,
+      limit;
 
     if (req.query.filter) {
       filter = JSON.parse(req.query.filter);
     }
 
+    if (query) {
```

Comment:
an empty object is truthy so this statement will always hit

Reply:
Maybe you could default it to null, or check to see if the object isn't empty?
Lodash has a good `isEmpty` function to use on objects.

Reply:
Good point.